app/src/androidTest/java/com/reach5/identity/sdk/demo/SandboxSuccessTest.kt

```diff
@@ -0,0 +1,198 @@
+package com.reach5.identity.sdk.demo
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
```

Comment:
This seems to be [deprecated](https://developer.android.com/reference/androidx/test/runner/AndroidJUnit4)